express/scripts/scripts.js

```diff
@@ -1121,8 +1121,8 @@ function displayEnv() {
     /* setup based on referrer */
     if (document.referrer) {
       const url = new URL(document.referrer);
-      if (url.hostname.endsWith('.adobeprojectm.com')) {
-        setHelixEnv('stage', { spark: url.hostname });
+      if (url.hostname.endsWith === '.adobeprojectm.com') {
```

Comment:
why not?

```suggestion
      if (url.hostname.endsWith('.adobeprojectm.com')) {
```

I don't think the condition as you wrote it will ever be true.